Obsidian/Obsidian.csproj

```diff
@@ -52,15 +53,22 @@
 
   <ItemGroup>
     <PackageReference Include="BouncyCastle.NetCoreSdk" Version="1.9.3.1" />
+    <PackageReference Include="CData.CockroachDB.EntityFrameworkCore" Version="21.0.7807" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.6" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="5.0.6" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.9" />
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.9" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.8.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="5.0.1" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="5.0.0" />
+    <PackageReference Include="MySql.EntityFrameworkCore" Version="5.0.3.1" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="5.0.6" />
     <PackageReference Include="sharpcompress" Version="0.27.1" />
     <PackageReference Include="SharpNoise" Version="0.12.1.1" />
     <PackageReference Include="SharpZipLib" Version="1.3.1" />
+    <PackageReference Include="System.Text.Json" Version="5.0.2" />
```

Comment:
Seems this was left in-

I added the library to do a quick comparison but it seems it kept the package ref even though I uninstalled it. I'll remove it manually. Moving to S.T.J may be worth considering but is outside of the scope of this change.